package.json

```diff
@@ -63,6 +63,7 @@
     "@types/gulp": "3.8.32",
     "@types/gulp-util": "^3.0.34",
     "@types/hammerjs": "^2.0.35",
+    "@types/inquirer": "^0.0.43",
```

Comment:
Done. We needed a wrapper script because npm scripts do not interpolate environment variables. This should be fine for now.

I will definitely look for ways to improve this, but should be good for an initial implementation.